desktop/start.ts

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env node
-const portfinder = require('portfinder');
+import {config} from 'dotenv';
+import portfinder from 'portfinder';
+
 const concurrently = require('concurrently');
```

Comment:
To use `import` for this lib it need to be updated, because [the fix](https://github.com/open-cli-tools/concurrently/pull/456) exists in higher lib version. Should I do it?

Reply:
But is this fix available in a release? It was merged on Feb 4 2024, but last release is `8.2.2` on Oct 19, 2023

Reply:
The import seems to work on `v8.2.2`. But it also require some updates in the code:

```
const {result} = concurrently(processes, {
    inputStream: process.stdin,
    prefix: 'name',

    // Like Harry Potter and he-who-must-not-be-named, "neither can live while the other survives"
    killOthers: ['success', 'failure'],
});

return result.then(
    () => process.exit(0),
    () => process.exit(1),
) as never;
```